src/ui/annotations.ts

```diff
@@ -1918,16 +1919,29 @@ export function UserLayerWithAnnotationsMixin<
                   const property = properties[i];
                   const label = document.createElement("label");
                   label.classList.add("neuroglancer-annotation-property");
+
+                  const nameWrapper = document.createElement("span");
+                  nameWrapper.classList.add("neuroglancer-annotation-property-name-wrapper");
+                  label.appendChild(nameWrapper);
+
+                  const { description } = property;
+                  if (description !== undefined && description) {
```

Comment:
if description is undefined if(description) is already false so doesn't need both checks